packages/react-components/src/AccessibilityScenarios/Accordion.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Accordion, AccordionItem, AccordionHeader, AccordionPanel } from '@fluentui/react-accordion';
+import { Scenario, APP_TITLE, APP_TITLE_SEPARATOR } from './utils';
+
+export const SimpleFormAccordionAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Simple Form Accordion">
+      <Accordion defaultOpenItems={'personalDetails'}>
+        <AccordionItem value="personalDetails">
+          <AccordionHeader>Personal details</AccordionHeader>
+          <AccordionPanel>
+            <label htmlFor="name">Name:</label>
+            <input type="text" id="name" name="name" />
+            <label htmlFor="email">Email:</label>
+            <input type="text" id="email" name="email" />
+            <div role="group" aria-label="Gender">
+              <input type="radio" id="male" name="gender" value="male" />
+              <label htmlFor="male">male</label>
+              <input type="radio" id="female" name="gender" value="female" />
+              <label htmlFor="female">female</label>
+            </div>
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem value="residence">
+          <AccordionHeader>Residence</AccordionHeader>
+          <AccordionPanel>
+            <label htmlFor="street">Street:</label>
```

Comment:
let's use the Fluent v9 `Label` component for this

Reply:
Okay, done.